WooCommerce/WooCommerceTests/ViewRelated/Settings/SettingsViewModelTests.swift

```diff
@@ -1,6 +1,7 @@
 import Codegen
 import XCTest
 import Yosemite
+import class Networking.UserAgent
```

Comment:
I think this can be removed:
```suggestion
```

Reply:
Yep, thank you. Must have been something else stopping it from building...